components/var_dumper/introduction.rst

```diff
@@ -73,6 +74,33 @@ current PHP SAPI:
     #. From time to time, run ``composer global update`` to have the latest
        bug fixes.
 
+
+
+Using the VarDumper Component in Your PHPUnit Test Suite
+--------------------------------------------------------
+
+The VarDumper component provides
+:class:`a trait <Symfony\\Component\\VarDumper\\Test\\VarDumperTestTrait>`
+that can help writting some of your tests for PHPUnit.
+By adding::
```

Comment:
What about:

```rst
You can use it by importing it into your test suite::

    use Symfony\Component\VarDumper\Test\VarDumperTestTrait;

    // ...
    class YourTest extends \PHPUnit_Framework_TestCase
    {
        use VarDumperTestTrait;
    }

This will provide you with two new assertions:

[...]
```